manifests/overlays/dev/kustomization.yaml

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+bases:
```

Comment:
`bases` is deprecated, use `resources` instead https://github.com/kubernetes-sigs/kustomize/issues/2243

`kustomize edit add base ../../base` will also use the `resources` field.